### PR TITLE
refactor: remove unused code

### PR DESCRIPTION
**Pull request by adekbadek**
Removes some unused code that was copied from `newspack-plugin` and paster in this repo in #2.  

## How to test

Smoke test – ensure the removed stuff is not used indeed

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -3,10 +3,6 @@ require( '@rushstack/eslint-patch/modern-module-resolution' );
 module.exports = {
 	extends: [ './node_modules/newspack-scripts/config/eslintrc.js' ],
 	ignorePatterns: [ 'dist/', 'node_modules/' ],
-	globals: {
-		newspack_urls: 'readonly',
-		newspack_aux_data: 'readonly',
```
Comment (dkoo) [inline .eslintrc.js:8]: According to the [failing job](https://app.circleci.com/pipelines/github/Automattic/newspack-multibranded-site/168/workflows/262de1b7-6cd3-4bc8-9607-f73aeb645134/jobs/675), if removing `newspack_aux_data` from this globals declaration, we need to include it as in a `/* global newspack_aux_data */` statement at the top of Brand.js.

**Review by dkoo — CHANGES_REQUESTED** [review]
Ran into one linting issue, otherwise looks good.

File: .eslintrc.js (hunk 1 above)
Comment (adekbadek) [reply .eslintrc.js:8]: Oops, added in caff08e02f859d9f4169c3cda00d8f71a0d401fb

**Review by dkoo — APPROVED** [review]

**Comment by chickenn00dle** [comment]
@adekbadek, it looks like these changes cause the multibranded site wizard to break with the following errors:

```
16:08:02.323 Uncaught ReferenceError: newspack_urls is not defined
    Qi admin.js:16797
    React 16
    <anonymous> admin.js:17936
    domReady dom-ready.js:71
    <anonymous> admin.js:17934
    <anonymous> admin.js:17939
    <anonymous> admin.js:17939
admin.js:16797:36
16:08:02.325 Uncaught TypeError: href.startsWith is not a function
    UnforwardedExternalLink components.js:58643
    React 16
    <anonymous> admin.js:17936
    domReady dom-ready.js:71
    <anonymous> admin.js:17934
    <anonymous> admin.js:17939
    <anonymous> admin.js:17939
components.js:58643:46
```

Seems to be related to the globals that were removed.

**Comment by dkoo** [comment]
Reverting in #59

**Comment by dkoo** [comment]
The missing globals and the JS fatals they caused have been fixed in #60 and merged in 10c3f17. Thanks again to @chickenn00dle for catching that!

**Pull request merged**